src/app/components/elements/ReplyEditor.jsx

```diff
@@ -144,7 +144,7 @@ class ReplyEditor extends React.Component {
     initForm(props) {
         const {isStory, type, fields} = props
         const isEdit = type === 'edit'
-        const maxKb = isStory ? 100 : 16
+        const maxKb = isStory ? 65 : 16
```

Comment:
you've changed the max size for a story from 100 -> 65 here... that's a significant change. Can you point to an issue or bug? If not please revert.

Reply:
Need to check if the backend has some firm limit, but currently the max block size (as voted on by witnesses) is 64kb so it would not be possible to publish a comment bigger than that.

Reply:
Exactly this. The limit should probably actually be 64, but I figured if it was a pasting issue that users would at least realize there is this limit. It would be misleading to currently allow a total TX size larger than 65,536 bytes.